packages/components/psammead-most-read/src/Item/index.jsx

```diff
@@ -81,56 +35,6 @@ const StyledLink = styled.a`
   }
 `;
 
-const StyledWrapper = styled.div`
-  @media (max-width: ${GEL_GROUP_0_SCREEN_WIDTH_MAX}) {
-    min-width: ${props =>
-      listHasDoubleDigits(props) ? doubleDigitWidth(props).group0 : 'auto'};
-  }
-
-  @media (min-width: ${GEL_GROUP_1_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
-    min-width: ${props =>
-      listHasDoubleDigits(props) ? doubleDigitWidth(props).group1 : 'auto'};
-  }
-
-  /* different number order for when css grid is not supported  */
-  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
-    min-width: ${props =>
-      columnIncludesDoubleDigits(props, false)
-        ? doubleDigitWidth(props).group2
-        : 'auto'};
-  }
-
-  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
-    min-width: ${props =>
-      columnIncludesDoubleDigits(props, false)
-        ? doubleDigitWidth(props).group3
-        : 'auto'};
-  }
-
-  @supports (${grid}) {
-    @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
-      min-width: ${props =>
-        columnIncludesDoubleDigits(props, true)
-          ? doubleDigitWidth(props).group2
-          : 'auto'};
-    }
-
-    @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
-      min-width: ${props =>
-        columnIncludesDoubleDigits(props, true)
-          ? doubleDigitWidth(props).group3
-          : 'auto'};
-    }
-  }
-
-  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
-    min-width: ${props =>
-      props.listIndex === 4 && listHasDoubleDigits(props)
-        ? doubleDigitWidth(props).group5
-        : 'auto'};
-  }
-`;
-
 const StyledItem = styled.div`
   padding-bottom: ${GEL_SPACING_TRPL};
   ${paddingStart}: ${GEL_SPACING_DBL};
```

Comment:
I know this isn't added in this PR, but isn't `paddingStart` a function that needs `{ dir }` passed into it? Also with the lines below.
```suggestion
  ${(props) => paddingStart(props)}: ${GEL_SPACING_DBL};
```

Reply:
yep that is true 👀

Reply:
This is currently working, since it is a function that is defined above on line 29 of this file https://github.com/bbc/psammead/pull/2869/files#diff-fae99f9f726c51a0ee2927a1c6e538feR20 The function *does* handle the props being passed in.

Reply:
Ah, are the props automatically passed into the function?

Reply:
@thekp's just showed me that it does 👍

Reply:
When using this component, the dir prop is passed in as a prop `<StyledItem dir={dir} ></StyledItem>` https://github.com/bbc/psammead/pull/2869/files#diff-fae99f9f726c51a0ee2927a1c6e538feR56